taskt/Core/Automation/Commands/IE Browser Commands/IEElementActionCommand.cs

```diff
@@ -1,53 +1,66 @@
-﻿using System;
+﻿using MSHTML;
+using SHDocVw;
+using System;
 using System.Collections.Generic;
 using System.Data;
 using System.Drawing;
 using System.Linq;
 using System.Windows.Forms;
 using System.Xml.Serialization;
-using Newtonsoft.Json;
-using OpenQA.Selenium;
-using taskt.UI.CustomControls;
-using taskt.UI.Forms;
-using SHDocVw;
-using System.Data;
-using MSHTML;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
 using taskt.Core.Automation.User32;
 using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("IE Browser Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to manipulate (get or set) elements within the HTML document of the associated IE web browser.  Features an assisting element capture form")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements the 'InternetExplorer' application object from SHDocVw.dll and MSHTML.dll to achieve automation.")]
+    [Group("IE Browser Commands")]
+    [Description("This command provides a number of functionalities or actions to perform Automation on Web Elements through the IE Browser.")]
     public class IEElementActionCommand : ScriptCommand
     {
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the instance name")]
+        [PropertyDescription("Instance Name")]
+        [InputSpecification("Specify name or variable for the IE Browser Instance to perform automation through.")]
```

Comment:
Change to "Specify the name or variable of the  IE Browser Instance to utilize for the automation."

Reply:
done